crmsh/xmlutil.py

```diff
@@ -675,6 +684,22 @@ def sanitize_cib(doc):
     xml_processnodes(doc, true, remove_text)
     xmltraverse(doc, drop_attr_defaults)
 
+def sanitize_cib_for_patching(doc):
+    """
+    Custom version of sanitize_cib which
+    doesn't sort container children, to use
+    for processing the original CIB when
+    generating a patch to apply using crm_diff.
+    """
+    xml_processnodes(doc, is_status_node, rmnodes)
+    # xml_processnodes(doc, true, printid)
+    # xml_processnodes(doc, is_emptynvpairs, rmnodes)
+    # xml_processnodes(doc, is_emptyops, rmnodes)
```

Comment:
I would maybe remove the commented lines to make it easier to read and visualize the difference between the two functions.

Reply:
alright, makes sense, I'll do that.